### PR TITLE
Fix simple Eslint issues in some tests

### PR DESCRIPTION
**Pull request by berndsi**
Also removed a few occurrences of unused variables when I noticed them.

### CODE REVIEW

**Comment by Pchelolo** [comment]
omg that's a lot :) I never had the time to eslint the test code, so it's just excluded from the linter, but this will make it much easier to actually enable eslint on test code. Thank you @berndsi 

Will wait for travis before merging.

File: test/features/buckets/key_value_bucket.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
-"use strict";
+'use strict';
+
+/* global describe, it, before */
```
Comment (Pchelolo) [inline test/features/buckets/key_value_bucket.js]: Actually, you can avoid doing that by placing a test-specific eslint config into the root of the test folder - I think it's much nicer like this. 

Example: https://github.com/wikimedia/preq/blob/master/test/.eslintrc.yml

**Comment by berndsi** [comment]
Not sure why the test is failing. I just ran it twice locally, and it was fine.
Added the .eslintrc.yml file, as you recommended.

**Comment by Pchelolo** [comment]
let's wait for it to run again, hopefully, they will fix themselves

**Comment by Pchelolo** [comment]
ok, bien. travis kinda pass

**Comment by Pchelolo** [comment]
Thank you @berndsi !

**Pull request merged**